.github/workflows/comprehensive-ci.yml

```
@@ -0,0 +1,535 @@
+          fi
+
+      - name: Run basic integration tests
+        run: |
+          echo "✅ Integration tests placeholder - implement based on your needs"
+          # Add your integration test commands here
+
+  # End-to-End Tests
+  e2e-tests:
+    runs-on: ubuntu-latest
+    needs: [integration-tests]
+    if: needs.setup_quality_gates.outputs.e2e_required == 'true'
```

actionlint (1.7.7)

property "setup_quality_gates" is not defined in object type {integration-tests: {outputs: {}; result: string}}

(expression)